ocaml/xapi/xapi_pool.ml

```diff
@@ -43,8 +43,16 @@ let rpc host_address xml =
       (Api_errors.Server_error
          (Api_errors.pool_joining_host_connection_failed, []))
 
+let get_pool ~rpc ~session_id =
+  match Client.Pool.get_all ~rpc ~session_id with
+  | [] ->
+      let err_msg = "Remote host does not belong to any pool." in
```

Comment:
"any pool" sounds like there could be many pools but can we have more than one pool?

Reply:
I can change it to "a pool". I wish this call would just return a single reference, always. Not sure why it's not this way.